Launcher/default.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs ? import <nixpkgs> { }
+, stdenv ? pkgs.stdenv
+, lib ? pkgs.lib
+, writeText ? pkgs.writeText
+, dotnet-sdk ? pkgs.dotnet-sdk
+, dotnetPackages ? pkgs.dotnetPackages
+, fetchurl ? pkgs.fetchurl
+, linkFarmFromDrvs ? pkgs.linkFarmFromDrvs
+}:
+
+let
+  fetchNuGet = { pname, version, sha256 }: fetchurl {
+    name = "${pname}-${version}.nupkg";
+    url = "https://www.nuget.org/api/v2/package/${pname}/${version}";
+    inherit sha256;
+  };
```

Comment:
It is currently recommended to use `hash` and not `sha256` for anything nixpkgs related 

https://nixos.org/manual/nixpkgs/stable/#fetchurl